### PR TITLE
Lucy/updated 53-59 + CAS

### PR DESCRIPTION
**Pull request by LucyWuu**
### Implementation description

- Updated member row 53 - 69 & Team Children's Aid Society

### Steps to test

1.

### What should reviewers focus on?

-

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: constants/members.json [hunk 1]
```diff
@@ -164,9 +164,9 @@
     {
       "name": "Eugene Zhang",
       "role": "developer",
-      "term": 1235,
-      "teams": ["casa"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fw23headshots%2Fdev_eugene_zhang.jpeg?alt=media&token=8b3bf4cc-95f0-4c16-8ac0-2c05bea159b3"
+      "term": 1236,
+      "teams": ["cas"],
```
Comment (Jess2772) [inline constants/members.json:168]: this is for childrens aid society right? The code still seems to be "casa" and not "cas", correct me if im wrong though

File: constants/members.json [hunk 2]
```diff
@@ -3429,6 +3429,76 @@
...
+      "role": "developer",
+      "term": 1236,
+      "teams": ["infrastructure"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_maisha_thasin.jpeg?alt=media&token=b4ddba89-ccd2-4345-b136-5b80475ca4a7"
+    },
+    {
+      "name": "Maisha Thasin",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["marillac-place"],
```
Comment (Jess2772) [inline constants/members.json:3472]: I don't see marillac-place added to the team ids at the top, i think youll need to add these

File: constants/members.json (hunk 1 above)
Comment (HeetShah) [reply constants/members.json:168]: @Jess2772 is correct. @LucyWuu my apologies, I should've been more specific in relating the team name with the one in the code

File: constants/members.json (hunk 2 above)
Comment (HeetShah) [reply constants/members.json:3472]: @Jess2772 is right. And let's use `mp` for Marillac Place  "teams"

**Review by HeetShah — CHANGES_REQUESTED** [review]
Think everything is right except for the comments left

File: constants/members.json (hunk 1 above)
Comment (LucyWuu) [reply constants/members.json:168]: @HeetShah no worries! I should've confirmed it with you. @Jess2772 thanks for pointing this out!

File: constants/members.json (hunk 2 above)
Comment (LucyWuu) [reply constants/members.json:3472]: gotchu

File: constants/members.json [hunk 3]
```diff
@@ -164,9 +176,9 @@
     {
       "name": "Eugene Zhang",
       "role": "developer",
```
Comment (Jess2772) [inline constants/members.json:178]: so Eugene is actually now technical lead for casa, could you change this?

**Review by Jess2772 — CHANGES_REQUESTED** [review]

File: constants/members.json [hunk 4]
```diff
@@ -3429,6 +3441,76 @@
...
+    },
+    {
+      "name": "Maisha Thasin",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["infra"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_maisha_thasin.jpeg?alt=media&token=b4ddba89-ccd2-4345-b136-5b80475ca4a7"
+    },
+    {
+      "name": "Maisha Thasin",
```
Comment (Jess2772) [inline constants/members.json:3481]: Guessing this should be for Shahan, but name is still Maisha

File: constants/members.json [hunk 5]
```diff
@@ -3429,6 +3441,76 @@
...
+    },
+    {
+      "name": "Janaki Menon",
+      "role": "vp talent",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVP%20Talent_Janaki_Menon.jpeg?alt=media&token=51ef769e-0970-44d5-b0ef-7051f920cddc"
+    },
+    {
+      "name": "Janaki Menon",
```
Comment (Jess2772) [inline constants/members.json:3502]: Same here, this should be for Vaaranan instead of Janaki

**Review by Jess2772 — CHANGES_REQUESTED** [review]

File: constants/members.json (hunk 4 above)
Comment (LucyWuu) [reply constants/members.json:3481]: @HeetShah can you help confirm?

File: constants/members.json (hunk 5 above)
Comment (LucyWuu) [reply constants/members.json:3502]: @HeetShah can you help confirm?

File: constants/members.json (hunk 4 above)
Comment (HeetShah) [reply constants/members.json:3481]: Looks like Maisha Thasin is listed as a developer for infra team. @LucyWuu this looks correct. @Jess2772 Shahan is technical lead for FCK. Did you mean someone/something else?

File: constants/members.json (hunk 3 above)
Comment (HeetShah) [reply constants/members.json:178]: confirmed

**Review by HeetShah — CHANGES_REQUESTED** [review]
Please check the comment for Eugene Zhang and push new changes 👍

File: constants/members.json (hunk 3 above)
Comment (LucyWuu) [reply constants/members.json]: updated

File: constants/members.json (hunk 5 above)
Comment (HeetShah) [reply constants/members.json:3502]: Yes, the `img` link is for `Vaaranan Yogalingam `. `Janaki` is the one above which you have right

File: constants/members.json (hunk 4 above)
Comment (Jess2772) [reply constants/members.json:3481]: Yea that one is fine. I was talking about lines 3474 and 3482, as they both have the same name field

**Review by HeetShah — APPROVED** [review]

**Pull request merged**